test-data/packet.yaml

```
@@ -195,3 +195,83 @@ code:
+      "r2.packet_offset=[-4098, 0]", "r2.svalue=[4098, 2147418112]", "r2.type=packet",
+      "r1.packet_offset=0", "r1.svalue=[4098, 2147418112]", "r1.type=packet"]
+
+code:
+  <start>: |
+    r1 -= r2
+    r3 = 101
+    assume r3 s<= r1;
+    r0 = *(u8 *)(r2 + 100);
+
+post: ["meta_offset=[-4098, 0]",
+  "r0.svalue=[0, 255]", "r0.type=number", "r0.uvalue=[0, 255]",
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 7 but found 2

(indentation)